### PR TITLE
Create lit-html-next-bug-report.md

### PR DESCRIPTION
**Pull request by arthurevans**
Add issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Actual Results
+<!-- Example: Error is thrown -->
+
+## Browsers Affected
+<!-- Check all that apply -->
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Edge
+- [ ] Safari 11
```
Comment (justinfagnani) [inline .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md:45]: This should just be `Safari` now, and let's remove the rest until we support them

File: .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+<!--
+Please read our contribution guidelines:
+  https://github.com/Polymer/lit-html/blob/master/CONTRIBUTING.md#filing-issues
+-->
+
+# Description
+A clear and concise description of what the bug is.
+
+# Steps to Reproduce
```
Comment (justinfagnani) [inline .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md:18]: Let's add a version number

```
# lit-html version
<!-- Check window.litHtmlVersions or npm ls lit-html -->
```

**Review by justinfagnani — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+name: lit-html lit-next preview bug report
+about: Create a report to help us improve
```
Comment (arthurevans) [inline .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md:3]:
```suggestion
about: Use this template to report an issue with the lit-html PREVIEW release (lit-html@next-major)
```

File: .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+<!--
+Please read our contribution guidelines:
+  https://github.com/Polymer/lit-html/blob/master/CONTRIBUTING.md#filing-issues
+-->
+
+# Description
+A clear and concise description of what the bug is.
+
```
Comment (arthurevans) [inline .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md:17]:
```suggestion

# lit-html version

<!-- Check window.litHtmlVersions or npm ls lit-html -->

```

File: .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md (hunk 1 above)
Comment (arthurevans) [inline .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md:45]:
```suggestion
- [ ] Safari
```

File: .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+## Actual Results
+<!-- Example: Error is thrown -->
+
+## Browsers Affected
+<!-- Check all that apply -->
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Edge
+- [ ] Safari 11
+- [ ] Safari 10
```
Comment (arthurevans) [inline .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md:46]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+- [ ] IE 11
```
Comment (arthurevans) [inline .github/ISSUE_TEMPLATE/lit-html-next-bug-report.md:47]:
```suggestion
```

**Comment by arthurevans** [comment]
Updated.

**Review by justinfagnani — APPROVED** [review]

**Pull request merged**